snoop/data/filesystem.py

```diff
@@ -153,8 +154,8 @@ def create_file(parent_directory, name, blob):
     create_directory_children(fake_root, archive_listing_data)
 
 
-@shaorma
-def digest(collection_pk, blob_pk, **depends_on):
+@shaorma('filesystem.digest')
+def digest(blob_pk, collection_pk, **depends_on):
```

Comment:
this should be `blob`, not the `blob_pk`, right? maybe the tests pass because Django's ORM does smart things with your `pk=Blob` filter